### ISSUE TITLE
Option to save content as PDF

### ISSUE THREAD

**Issue by kieranclancy** [susam/mathb#4]
Not sure if this is possible, but it would be great to be able to download a PDF of the math content. Perhaps with a "download PDF" link?

I realise this would require a lot of work with jsPDF or similar, but it's a nice idea for the future.

**Comment by janbok** [comment]
I think that this should be no big issue since the MathBin uses pandoc.

**Comment by susam** [comment]
Yes, it is possible to do this. However, I have kept the development on hold for a long time because I have been planning to rewrite the software in Python. Once that is done, I will be resuming with more development on this software.

**Comment by susam** [comment]
With issue #3 now resolved, it is now possible to save a PDF using the web browser's print preview mode. To try it out, visit https://mathb.in/1 first, then select File > Print from the web browser menu, and then select the option to save as PDF if your web browser provides such an option.
